### ISSUE TITLE
ordered-imports not failing when there is a blank line

### ISSUE THREAD

**Issue by alan-agius4** [palantir/tslint#3727]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.7.2

#### TypeScript code being linted

```ts
import { CommonModule } from "@angular/common";
import { NgModule } from "@angular/core";

import { IonicModule } from "ionic-angular";

import { SkeletonComponent } from "./skeleton/skeleton.component";
```

with `tslint.json` configuration:

```json
"ordered-imports": [
	true,
	{
		"grouped-imports": true
	}
],
```

#### Actual behavior
No error

#### Expected behavior
I expected that that there is an error that imports are not group. In fact if I run the linter with fixer (If i have an error in the imports) it will remove the extra blank line.

**Comment by timc13** [comment]
I'm quite confused as to why this was classified as a bug.

This seems to be intended behavior per the [documentation](https://palantir.github.io/tslint/rules/ordered-imports/):

> Groups of imports are delineated by blank lines. You can use these to group imports however you like, e.g. by first- vs. third-party or thematically or can you can enforce a grouping of third-party, parent directories and the current directory.

also see: https://github.com/palantir/tslint/issues/4394

**Comment by ericanderson** [comment]
@timc13 Thanks for chiming in. Upon reflection, I think I agree with you. Would you prefer we rollback the merge that forces grouping and retry with an extra flag for this case?
